docs/cli/cli-spans.md

```diff
@@ -0,0 +1,38 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: CLI
+--->
+
+# Semantic Conventions for CLI (Command Line Interface) programs
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions to apply when instrumenting CLI programs.
+operations to attributes on a Span.
```

Comment:
Nit typo

```suggestion
This document defines semantic conventions to apply when instrumenting CLI programs'
operations to attributes on a Span.
```

Reply:
I think the second line was a copy and paste error, reworked the text a bit.